br/pkg/checksum/executor_test.go

```diff
@@ -108,4 +109,12 @@ func TestChecksum(t *testing.T) {
 		}
 		return nil
 	}))
+
+	exe4, err := checksum.NewExecutorBuilder(tableInfo3, math.MaxUint64).Build()
+	require.NoError(t, err)
+	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/br/pkg/checksum/checksumRetryErr", `return(true)`))
+	failpoint.Disable("github.com/pingcap/tidb/br/pkg/checksum/checksumRetryErr")
```

Comment:
```suggestion
	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/br/pkg/checksum/checksumRetryErr", `1*return(true)`))
```

Perhaps use the internal syntax to restrict how many time the failpoint being triggered? (See the SYSCTL_VARIABLES [here](https://man.freebsd.org/cgi/man.cgi?query=fail&sektion=9&apropos=0&manpath=FreeBSD%2B10.0-RELEASE) paragraph for more details)

Reply:
OK，I 'll update it.